operators/aqua/1.0.2/manifests/aqua-operator.v1.0.2.clusterserviceversion.yaml

```diff
@@ -4,12 +4,6 @@ metadata:
   name: aqua-operator.v1.0.2
   namespace: placeholder
   annotations:
-    # Setting operatorhub.io/ui-metadata-max-k8s-version annotation automatically
-    # The following is an informative annotation to let its users know that this distribution
-    # uses removed APIs in 1.22 and will not work on K8S clusters 1.22+.
-    # Following the findings for this distribution:
-    # this bundle is using APIs which were deprecated and removed in v1.22. More info: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22. Migrate the API(s) for CRD: (["aquacsps.operator.aquasec.com" "aquadatabases.operator.aquasec.com" "aquaenforcers.operator.aquasec.com" "aquagateways.operator.aquasec.com" "aquascanners.operator.aquasec.com" "aquaservers.operator.aquasec.com"])
-    operatorhub.io/ui-metadata-max-k8s-version: "1.21"
```

Comment:
Same @yossig-aquasec

Reply:
Thanks its old version that not supported 1.22